src/main/java/kr/codesqaud/cafe/controller/ArticleController.java

```diff
@@ -37,14 +38,25 @@ public String welcomePage(Model model) {
         return "welcome/index";
     }
 
-    @PostMapping("/qna/questions") // TODO : Username 가져오기
-    public String addArticle(@ModelAttribute Article article, BindingResult bindingResult, Model model) {
+    @GetMapping("/qna/qna_form")
+    public String writeArticle(HttpSession session, Model model) {
```

Comment:
```suggestion
    public String writeArticleForm(HttpSession session, Model model) {
```
메소드 이름에서 Form 이라는 걸 명시하면 좋을 것 같아요. POST /article 이랑 헷갈렸습니다.